cmake/Dependencies.cmake

```diff
@@ -464,6 +501,14 @@ if (USE_METAL)
   endif()
 endif()
 
+
+if (USE_ARM_COMPUTE)
```

Comment:
This should be above the code that imports arm compute library targets from Scons. The motivation is to check only `if (USE_ARM_COMPUTE)`, so if we need to enable this backend on other platforms, we need to patch only one snippet.